nautobot/virtualization/api/serializers.py

```diff
@@ -252,3 +254,34 @@ def validate(self, data):
                 )
 
         return super().validate(data)
+
+
+class VMInterfaceSerializerVersion13(TaggedObjectSerializer, ValidatedModelSerializer, StatusModelSerializerMixin):
```

Comment:
Same suggestion here about `VMInterfaceSerializer` and `VMInterfaceSerializerVersion12`.